22,29c22,29
< RESULT[0], LEDR[0] 
< RESULT[1], LEDR[1] 
< RESULT[2], LEDR[2] 
< RESULT[3], LEDR[3] 
< RESULT[4], LEDR[4] 
< RESULT[5], LEDR[5] 
< RESULT[6], LEDR[6] 
< RESULT[7], LEDR[7] 
---
> RESULT[0], LEDR[0]
> RESULT[1], LEDR[1]
> RESULT[2], LEDR[2]
> RESULT[3], LEDR[3]
> RESULT[4], LEDR[4]
> RESULT[5], LEDR[5]
> RESULT[6], LEDR[6]
> RESULT[7], LEDR[7]
--- correct_outputs/lookahead_adder_subtractor_8_bit.csv	2016-03-12 23:30:40.010456681 -0500
+++ generated_outputs/lookahead_adder_subtractor_8_bit.csv	2016-03-12 23:49:44.500522999 -0500
@@ -19,13 +19,13 @@
 
 MODE, SW[17]
 
-RESULT[0], LEDR[0] 
-RESULT[1], LEDR[1] 
-RESULT[2], LEDR[2] 
-RESULT[3], LEDR[3] 
-RESULT[4], LEDR[4] 
-RESULT[5], LEDR[5] 
-RESULT[6], LEDR[6] 
-RESULT[7], LEDR[7] 
+RESULT[0], LEDR[0]
+RESULT[1], LEDR[1]
+RESULT[2], LEDR[2]
+RESULT[3], LEDR[3]
+RESULT[4], LEDR[4]
+RESULT[5], LEDR[5]
+RESULT[6], LEDR[6]
+RESULT[7], LEDR[7]
 
 OVERFLOW, LEDR[17]
--- correct_outputs/lookahead_adder_subtractor_8_bit.csv	2016-03-12 23:52:32.860532755 -0500
+++ generated_outputs/lookahead_adder_subtractor_8_bit.csv	2016-03-13 00:10:35.670595499 -0500
@@ -1,31 +0,0 @@
-To, Location
-A[0], SW[0]
-A[1], SW[1]
-A[2], SW[2]
-A[3], SW[3]
-A[4], SW[4]
-A[5], SW[5]
-A[6], SW[6]
-A[7], SW[7]
-
-B[0], SW[8]
-B[1], SW[9]
-B[2], SW[10]
-B[3], SW[11]
-B[4], SW[12]
-B[5], SW[13]
-B[6], SW[14]
-B[7], SW[15]
-
-MODE, SW[17]
-
-RESULT[0], LEDR[0]
-RESULT[1], LEDR[1]
-RESULT[2], LEDR[2]
-RESULT[3], LEDR[3]
-RESULT[4], LEDR[4]
-RESULT[5], LEDR[5]
-RESULT[6], LEDR[6]
-RESULT[7], LEDR[7]
-
-OVERFLOW, LEDR[17]
